src/components/Trade/hooks/useSwapper/useSwapper.tsx

```diff
@@ -40,25 +38,35 @@ export const useSwapper = () => {
 
   // Selectors
   const flags = useSelector(selectFeatureFlags)
-  const assetIds = useSelector(selectAssetIds)
+  const assetsByMarketCapById = useSelector(selectSortedAssetsById)
 
   // Hooks
-  const [swapperManager, setSwapperManager] = useState<SwapperManager>(() => new SwapperManager())
+  const [swapperManager, setSwapperManager] = useState<SwapperManager>()
```

Comment:
curious on @0xApotheosis thoughts here as i'm not sure what the impact will be

Reply:
We fallback to an empty array if `swapperManager` is ever `undefined` where it's used inside `useSwapper`, so this should be fine, and gives us a nice little performance boost - good catch big wood.